src/main/java/zd/zero/waifu/motivator/plugin/assets/AudibleAssetDefinitionService.kt

```diff
@@ -0,0 +1,34 @@
+package zd.zero.waifu.motivator.plugin.assets
+
+import kotlin.random.Random
+
+object AudibleAssetDefinitionService {
+
+    private val ranbo = Random(System.currentTimeMillis())
```

Comment:
question: Is there a reason why it is named `ranbo`? Or is it just a typo? I do not understand what it means and did a quick search if there's a convention for random.

Reply:
![Ranbo](https://upload.wikimedia.org/wikipedia/en/a/a3/Rambo3poster.jpg)

:joy: 

No there is not a good reason.

Reply:
Noooooooooooo 😂

Reply:
_I'll fix it_ 

![No Fun Allowed](https://i.kym-cdn.com/entries/icons/original/000/015/266/Z7HeRxU.png)